addons/website_event_track/views/event_track_templates_agenda.xml

```diff
@@ -154,6 +154,7 @@
             </t>
         </table>
         </div>
+        <span class="o_we_online_agenda_overlay py-5 px-3 pe-none z-index-1"/>
```

Comment:
I guess this isn't done yet?

Reply:
It's done actually :) It's the overlay element that will be triggered in JS to highlight the fact that the table is "scrollable"